core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -101,17 +101,26 @@ Function Get-Java-Opts {
 	return $JAVA_OPTS
 }
```

Comment:
"standalone.bat + jdk11 + gc enabled" creates:
* standalone\log\gc.log
* standalone\log\gc.log.0

"standalone.ps1 + jdk11 + gc enabled" creates:
* standalone\log\gc.log

What causes this different behavior?

Reply:
I kind of thought it was a weird JVM issue. It noticed it was some times created. The PowerShell script was the only odd one out.

Reply:
sh and bat files creates two log files, so I assume that this (two gc log files) is correct behaviour.

PowerShell script should start WF correctly. PowerShell script should start WF in the same way as bat and sh script.

So this is a bug and this needs to be fixed.

Reply:
The only difference I was an extra quote. On the 3rd boot of the `standalone.ps1` the `gc.log.0` shows up for some reason.

```-Xlog:gc*:file="\"C:\Users\jperkins\projects\wildfly-core\build\target\wildfly-core-7.0.0.Alpha2-SNAPSHOT\standalone\log\gc.log\"":time,uptimemillis:filecount=5,filesize=3M
```
```
-Xlog:gc*:file=\"C:\Users\jperkins\projects\wildfly-core\build\target\wildfly-core-7.0.0.Alpha2-SNAPSHOT\standalone\log\gc.log\":time,uptimemillis:filecount=5,filesize=3M
```

The extra quote is in the `.bat` startup though. I'm honestly not sure the issue is.